docs/using-connector.md

```diff
@@ -75,6 +77,11 @@ conf.set("spark.riak.connections.max", "50")
 sc = pyspark.SparkContext("spark://127.0.0.1:7077", "test", conf)
 ```
 
+## Failover Handling
+
+When reading data from Riak, the Spark-Riak connector will run a full bucket query. This query first obtains metadata, called a coverage plan, from Riak about where the data is located in the cluster. This coverage plan will have a list of nodes were the data and its replicas live, but the Spark-Riak connector will use the first entry in this list to read the data from Riak. After the Spark-Riak connector recieves the coverage plan, a node may fail and the reads going to the node will also fail. In this event, the Spark-Riak connector will request and alternative coverage plan from Riak for the data that failed to be read from Riak. This alternative coverage plan will return the next node that holds the data or its replicas. Then the Spark-Riak connector will attempt to read the missing data from the nodes in the alternative coverage plan. This will continue until the missing data is successfully read or the list of alternative nodes has been exhausted.
```

Comment:
> This coverage plan will have a list of nodes were the data and its replicas live
This sentence is not totally correct, because coverage plan doesn't contain information about replicas. That's why we have to execute additional coverage plan query in case of fail. Let's remove mention of replicas from this sentence.